test/Dapr.AspNetCore.IntegrationTest.App/Startup.cs

```diff
@@ -46,10 +46,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
                 endpoints.MapPost("/topic-a", context => Task.CompletedTask).WithTopic("A");
 
-                endpoints.MapPost("/routingwithstateentry/{widget}", async context =>
+                endpoints.MapPost("/routingwithstateentry/{testStore}/{widget}", async context =>
```

Comment:
Same comment as the controller, you don't need to add `{testStore}` to the route, it's not doing anything meaningful in the test.